libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1665,6 +1668,33 @@ bool AP_Arming::blending_auto_switch_checks(bool report)
 }
 #endif
 
+#if AP_CRASHDUMP_ACK_ENABLED
+bool AP_Arming::crashdump_checks(bool report)
+{
+    int fd = AP::FS().open("@SYS/crash_dump.bin", O_RDONLY);
```

Comment:
better to call hal.util->last_crash_dump_size() and check > 0, faster and less dependencies

Reply:
Done